aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -514,6 +514,137 @@ Tensor& range_out(Tensor& result, Scalar start, Scalar end, Scalar step) {
   return at::_th_range_out(result, start, end, step);
 }
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ triangle ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tensor tril_indices(
+    int64_t row, int64_t col, int64_t offset, const TensorOptions& options) {
```

Comment:
These TensorOptions& aren't being used right now (are you planning on using them later?)

Reply:
Actually, I realized I don't understand how TensorOptions works, let me look into that a little